src/webgpu/api/operation/rendering/draw.spec.ts

```diff
@@ -1,15 +1,27 @@
 export const description = `
 Tests for the general aspects of draw/drawIndexed/drawIndirect/drawIndexedIndirect.
 
-TODO: plan and implement
-- draws (note bind group state is not tested here):
-    - various zero-sized draws
-    - draws with vertexCount not aligned to primitive topology (line-list or triangle-list) (should not error)
-    - index buffer is {unset, set}
-    - vertex buffers are {unset, set} (some that the pipeline uses, some it doesn't)
-      (note: to test this, the shader in the pipeline doesn't have to actually use inputs)
-    - x= {draw, drawIndexed, drawIndirect, drawIndexedIndirect}
-- ?
+TODO:
```

Comment:
I think this basically covers the stuff in indexed_draw.spec.ts as well. The additional stuff in there has to do with OOB behavior which should go in one of the robust_access tests?

Reply:
Deleted this file.